Book-A-Desk.Api/Book-A-Desk.Api.fsproj

```diff
@@ -19,8 +19,12 @@
     <Compile Include="DynamoDB.fs" />
     <Compile Include="Program.fs" />
     <Content Include="docs\bookADeskOpenAPISpecification.json" />
-    <Content Include="appsettings.Development.json" />
```

Comment:
I don't think anyone uses (or should use) appsettings anymore. You can add variables in docker compose if you need to add them in your environment

Reply:
in the ui, we use a local .env file, we can probably use one to pass environment variables to the application